vueui/src/assets/global.scss

```diff
@@ -15,6 +16,41 @@ body {
     margin: 0px;
     padding: 8px;
     font-family: Arial, Helvetica, sans-serif;
+    overflow-y: hidden;
+    overflow-x: auto;
+    min-width: 240px;
+}
+
+#content {
+    height: calc(100vh - 36px);
+    &.mcomputer {
+        height: calc(100vh - 72px);
+    }
+    &.uiDebug {
+        height: calc(50vh - 36px);
+        &.mcomputer {
+            height: calc(50vh - 72px);
+        }
+    }
+}
```

Comment:
Please refrain from using `calc` as IE badly supports it. See https://caniuse.com/?search=calc

Reply:
Shouldn't be an issue in this case, the bad support isn't in simple cases like this thankfully. But I agree we have to be cautious about it.